src/doc/src/guide/project-layout.md

```diff
@@ -48,6 +48,8 @@ files, place a `main.rs` file along with the extra [*modules*][def-module]
 within a subdirectory of the `src/bin`, `examples`, `benches`, or `tests`
 directory. The name of the executable will be the directory name.
 
+Binaries, examples, benches and integration tests follow `kebab-case` naming style. Modules within those targets are `snake_case` following the [Rust standard](https://rust-lang.github.io/rfcs/0430-finalizing-naming-conventions.html).
```

Comment:
This is likely best left to a style guide, with us finding an appropriate place to link out to it.

Reply:
Is there a style guide that makes sense to use?

Reply:
The way I see it, right now no one in the rust ecosystem directly says that build targets are `kebab-case`. This document is the closest but it is not clear.